app/components/footer_component.html.erb

```diff
@@ -82,7 +82,6 @@
     </div>
     <div class="site-footer-bottom">
       <div class="site-footer-bottom__links-container">
-        <%= link_to("Search", search_path) %>
```

Comment:
Could you possibly shift the A-Z index so it appears first in the list (replacing Search) rather than last @martyn-w ? Thanks!